### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by bentleybiscuit**

### CODE REVIEW

File: navbarBuild.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	float: left;
+	list-style: none;
+	width: 33%;
+}
+
+nav li:hover {
+	color: black;
+}	
+
+
```
Comment (jdselby24) [inline navbarBuild.css]: consider adding a media query to shrink the text to better fit mobile screen sizes

File: navbarBuild.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+nav {
+	background-color: #019dcf;
+	color: #ffffff;
+	font-size: 20px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	padding: 40px 60px;
+	overflow: auto;
```
Comment (jdselby24) [inline navbarBuild.css]: overflow auto should be on ul not nav

File: navbarBuild.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	
+}
+
+
+nav ul {
+	margin: 0;
+	width: 100%;
+	padding: 0;
+	text-align: center;
```
Comment (jdselby24) [inline navbarBuild.css]: add overflow auto here

**Review by jdselby24 — CHANGES_REQUESTED** [review]
just a few little things, otherwise all good

File: navbarBuild.css (hunk 3 above)
Comment (bentleybiscuit) [reply navbarBuild.css]: Thanks Josh. Could you review the change to font sizing thanks.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+	color: #ffffff;
+	font-size: 20px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	padding: 40px 60px;
+	
+}
+
+
+@media screen and (max-width: 500px){
```
Comment (jdselby24) [inline styles.css]: looks good 👍 , but i believe its good practice to place media queries under the rest of the regular css

**Review by jdselby24 — CHANGES_REQUESTED** [review]

**Comment by sudokufan** [comment]
did you mean to merge this yesterday?

**Pull request merged**